### PR TITLE
Include Content.ad in Supported ad networks list

### PR DESCRIPTION
**Pull request by jlucero-contentad**
see #4835

### CODE REVIEW

**Comment by lannka** [comment]
LGTM. will merge once it's green

**Comment by jlucero-contentad** [comment]
Thanks @lannka. My Travis CL build test passed.

**Comment by jlucero-contentad** [comment]
@lannka the Content.ad link is broken: it goes to the wrong file path. What did I do wrong?
[https://www.ampproject.org/docs/reference/amp-ad.html](https://www.ampproject.org/docs/reference/amp-ad.html)

File: extensions/amp-ad/amp-ad.md [hunk 1]
```diff
@@ -89,6 +89,7 @@ resources in AMP. It requires a `type` argument that select what ad network is d
 - [Atomx](../../ads/atomx.md)
 - [Chargeads](../../ads/chargeads.md)
 - [Colombia](../../ads/colombia.md)
+- [Content.ad](../ads/contentad.md)
```
Comment (lannka) [inline extensions/amp-ad/amp-ad.md]: @jlucero-contentad seems this is the cause. it missed extra `../`. No worry, I can fix that for you.

Comment (jlucero-contentad) [inline extensions/amp-ad/amp-ad.md]: Ah! I missed that. Thank you so much for fixing it @lannka

Comment (lannka) [inline extensions/amp-ad/amp-ad.md]: My fault didn't catch it in the first place. Fixed in #4992 . Please check back tmr :-)

Comment (lannka) [inline extensions/amp-ad/amp-ad.md]: @jlucero-contentad FYI, I verified the fix.

Comment (jlucero-contentad) [inline extensions/amp-ad/amp-ad.md]: @lannka thanks! I see the link is working now.

**Pull request merged**